spec/rails_helper.rb

```diff
@@ -3,14 +3,18 @@
 require File.expand_path("../../config/environment", __FILE__)
 require 'rspec/rails'
 require 'capybara/rails'
-require 'helpers/features_spec_helper'
+
+# require all helper files in helpers folder
+Dir[Rails.root.join('spec/helpers/**/*.rb')].each { |f| require f }
```

Comment:
Nice.
That is said: what do you think about the idea of adding this very folder to the support folder?
Doing so, you'll be able to have a dedicated `helpers` folder, in order to test... your Rails helpers. 😎